src/main/resources/application.properties

```diff
@@ -81,7 +81,7 @@ swymed-edituser-url=swymed-base-url/SwymedWebApi/api/Contact
 
 calibrationPageSize=5
 
-
+biological-screening-device-url=http://localhost:8085/ezdx-hub-connect-srv

```

Comment:
Not sure what this service is about.
Pretty sure we don't run a service at port 8085 along with widfly.
best to keep this in environment properties.